Automattic/wp-calypso
# Gutenboarding: Always enroll in FSE when creating a site

#### Changes proposed in this Pull Request

General clean-up of Gutenboarding, removing everything related to FSE opt-in and forcing `enable_fse` in the new site request `options` object.

It also lets the user pick a classic AnchorFM theme if necessary, following the discussion in p1642621207111500-slack-CHN6J22MP.

Closes #60253.

#### Testing instructions

Create a site through `/new`. You'll notice only Universal Themes show up in the design picker and that you're FSE eligible as soon as the site is created.

Creating a site by having [AnchorFM as a referer](pdgAPm-1G-p2) displays only Anchor classic themes in the design picker, even though the site will get enrolled in FSE.

#### Next steps

Merge [this commit](https://github.com/Automattic/wp-calypso/commit/28c45a937dd52f6f72ffb8127bd58e9375724834) putting Anchor themes at the top while displaying Universal Themes below them. Reference:  p1642633206135400/1642621207.111500-slack-CHN6J22MP
